features/step_definitions/logging_metrics.rb

```diff
@@ -681,6 +681,7 @@
           # for metering we need to create the
           @result = user.cli_exec(:create, f: "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/metering/default-storageclass-values.yaml")
           step %Q/all metering related pods are running in the "#{target_proj}" project/
+          step %Q/I wait for the "openshift-metering" metering to appear/
```

Comment:
does it make sense here to call `metering service has been installed successfully` instead of repeating the check in that other step?

Reply:
ok, I see the step will also install metering if missing, ignore